pkg/operator/k8sutil/configmap.go

```diff
@@ -63,7 +63,12 @@ func GetOperatorSetting(context context.Context, clientset kubernetes.Interface,
 
 func GetValue(data map[string]string, settingName, defaultValue string) string {
 	if settingValue, ok := data[settingName]; ok {
-		logger.Infof("%s=%q (configmap)", settingName, settingValue)
+		// Suppress logging of ROOK_WATCH_FOR_NODE_FAILURE setting
+		if settingName == "ROOK_WATCH_FOR_NODE_FAILURE" {
+			logger.Debugf("%s=%q (configmap)", settingName, settingValue)
```

Comment:
Shouldn't we log as info at least for the first time? if not we might not be able to check the setting value in operator logs like other configurations in production clusters.

Reply:
if we want to print at least once, we already know that the default value is true and if some users change the setting, it will be mostly in configmap or rarely in env var so we can get value from there.